feature/home/src/main/kotlin/com/nexters/bandalart/android/feature/home/ui/EmojiPickerUI.kt

```diff
@@ -0,0 +1,158 @@
+      "❤️", "🧡", "💛", "💚", "💙", "❤️‍🔥",
+    )
+
+    Column(
+      modifier = modifier
+        .fillMaxWidth().wrapContentHeight()
+        .background(White),
+    ) {
+      Text(
+        modifier = Modifier
+          .align(Alignment.End)
+          .padding(
+            end = 20.dp,
+            top = if(isBottomSheet) 16.dp else 0.dp,
+          )
+          .clickable {
+            emojiPickerScope
+              .launch { emojiPickerState.hide() }
+              .invokeOnCompletion {
+                if (!emojiPickerState.isVisible) onResult(selectedEmoji, false)
+              }
+          },
+        text = "완료",
+        color = Gray900,
+        fontFamily = pretendard,
+        fontWeight = FontWeight.W700,
+        fontSize = 16.sp.nonScaleSp,
+      )
+      var emojiIndex = 0
+      Column(
+        modifier = Modifier
+          .fillMaxWidth()
+          .padding(
+            top = if(isBottomSheet) 15.dp else 0.dp,
+            start = if(isBottomSheet) 15.dp else 0.dp,
+            end = if(isBottomSheet) 23.dp else 8.dp,
+            bottom = if(isBottomSheet) 26.dp else 0.dp,
+          ),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.SpaceEvenly,
+      ) {
+        repeat(4) {
+          Row(
+            modifier = Modifier
+              .fillMaxWidth()
+              .padding(top = 8.dp),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceEvenly,
+          ) {
+            repeat(6) {
+              val emojiItem = emojiList[emojiIndex++]
+              Card(
+                modifier = Modifier
+                  .padding(start = 8.dp)
+                  .weight(1f),
+                shape = RoundedCornerShape(12.dp),
+                border = when (emojiItem) {
+                  selectedEmoji -> {
+                    BorderStroke(
+                      1.dp,
+                      Gray400,
+                    )
+                  }
+                  prevSelectedEmoji -> {
+                    BorderStroke(
+                      1.dp,
+                      Color.Transparent,
+                    )
+                  }
+                  else -> null
+                },
+              ) {
+                Box(
+                  modifier = Modifier
+                    .fillMaxWidth()
+                    .aspectRatio(1f)
+                    .background(Gray100)
+                    .clickable {
+                      if (prevSelectedEmoji == emojiItem) selectedEmoji = ""
+                      else {
+                        prevSelectedEmoji = selectedEmoji
+                        selectedEmoji = emojiItem
+                      }
+                    },
+                  contentAlignment = Alignment.Center,
+                ) {
+                  EmojiText(
+                    emojiText = emojiItem,
+                    fontSize = 24.sp.nonScaleSp,
```

Comment:
```suggestion
fontSize = 24.sp
```
EmojiText 컴포저블 내부에서 이미 nonScaleSp 처리해주고 있어

Reply:
오키 뺄게!